bower.json

```diff
@@ -3,5 +3,10 @@
   "main": [
     "src/js.cookie.js"
   ],
-  "ignore": []
+  "ignore": [
+    "travis.sh",
+    "test",
```

Comment:
I'm not sure i understand this. Why would you want to integrate a third parties tests?